tools/export.py

```diff
@@ -12,6 +12,8 @@
     >>> python tools/export.py --model_name dbnet_resnet50 --data_shape 736 1280
     >>> # Export mindir of model `dbnet_resnet50` by loading local ckpt
     >>> python tools/export.py --model_name dbnet_resnet50 --data_shape 736 1280 --local_ckpt_path /path/to/local_ckpt
+    >>> # Export mindir of model whose architecture is defined by crnn_resnet34.yaml with local checkpoint
```

Comment:
can also update the export usage here: https://github.com/mindspore-lab/mindocr/tree/main/configs#model-export

Reply:
updated